packages/connectivity/src/scp-cf/destination/destination-from-registration.ts

```diff
@@ -70,9 +70,21 @@ export type DestinationWithName = Destination & { name: string };
 export function searchRegisteredDestination(
   options: DestinationFetchOptions
 ): Destination | null {
+  let decodedJwt;
+  // An error could be thrown if no jwt and no xsuaa service exist.
```

Comment:
[q] What do you mean "cloud be thrown"? Could implies to me that there is non-deterministic behavior here (which I don't expect). It should be "will be" if it's implemented like this or "should be" if it is a TODO.